src/tests/addresses.jsx

```diff
@@ -59,15 +54,24 @@ class ConfirmMultisigAddressTest extends Test {
       network: this.params.network,
       bip32Path: this.params.bip32Path,
       multisig: this.params.multisig,
+      addressIndex: this.params.addressIndex,
+      // probably shouldn't be necessary but it's here for wallet registration
+      name: this.params.description,
+      // only used with ledgers, version 2.1 and above
+      policyHmac: this.params.policyHmac,
     });
   }
 }
 
 const addressTests = (keystore) =>
   TEST_FIXTURES.multisigs.map((fixture) => {
+    // todo, this should be explicit, coming from the fixture
+    const addressIndex = fixture?.bip32Path.split("/").slice(-1)[0];
```

Comment:
Not sure I understand the todo, isn't that what you're doing?

Reply:
Yeah, I'm pulling this out of the fixture though and I think the fixture should be more explicit. BIP32 paths can be to a lot of things and it's not really a guarantee that the path is the full path to the address. I just know that it happens to be.

Reply:
Do you want to check for a min and max path length?

Reply:
Can't, at least not based on the standard we want to start enforcing. There could be any number of intermediate paths (unchained adds one for example for the wallet keys). Really the best way to do it (imo) is be explicit about it in the fixture. This is similar to how the Ledger api does it in fact where it expects just a single int that indicates the address index rather than inferring from some path.